Clone-a-specific-branch.md

```diff
@@ -0,0 +1,7 @@
+<h1>Clone a specific branch</h1>
+There are many cases where we want to clone a particular branch on oue loacal machine. For that we need to follow these methods:
+
+git clone -b <branch_name> <remote_repo>
```

Comment:
Enclose code/commands with back ticks for readability:
 `git clone -b <branch_name> <remote_repo>`